code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -413,7 +413,16 @@
 	update_icon()
 	return 1
 
+
 //This proc is called when you want to place an item into the storage item.
+//Its a safe proc for adding things to the storage that does the necessary checks. Object will not be moved if it fails
+/obj/item/weapon/storage/proc/add_to_storage(obj/item/W as obj, var/prevent_messages = 1)

```

Comment:
I feel like it should have the word "insert" in the proc name. To fall in line with the other two procs.